gitium/functions.php

```diff
@@ -206,10 +206,50 @@ function gitium_commit_and_push_gitignore_file( $path = '' ) {
 	gitium_merge_and_push( $commit );
 }
 
+if ( ! function_exists( 'gitium_before_merge_and_push' ) ) :
+	function gitium_before_merge_and_push() {
+		global $gitium_lock_path, $gitium_lock_handle;
+
+		$gitium_lock_path   = apply_filters( 'gitium_lock_path', ABSPATH . 'wp-content/.gitium-lock' );
```

Comment:
I think a better option would be to put the lock file into `.git` folder, eg. `.git/gitium-lock`.